### PR TITLE
on the app

### PR DESCRIPTION
**Pull request by AndreiD**
a little bit of reskin on the sample app. most notably retrofit 2.0...

### CODE REVIEW

File: app/src/main/java/com/hardsoftstudio/rxflux/sample/core/GitHubApi.java [hunk 1]
```diff
@@ -2,21 +2,51 @@
...
+    private static GitHubApi instance;
+
+    private static void create() {
+
+      //------- enable logging ------
+      OkHttpClient client = new OkHttpClient();
+      HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
+      interceptor.setLevel(HttpLoggingInterceptor.Level.BASIC);
+
+      //need custom headers ?
```
Comment (marcelpinto) [inline app/src/main/java/com/hardsoftstudio/rxflux/sample/core/GitHubApi.java:34]: Remove commented code please

File: app/src/main/java/com/hardsoftstudio/rxflux/sample/core/GitHubApi.java [hunk 2]
```diff
@@ -2,21 +2,51 @@
...
+  @GET("/repositories") Observable<ArrayList<GitHubRepo>> getRepositories();
+
+  @GET("/users/{id}") Observable<GitUser> getUser(@Path("id") String userId);
+
+  class Factory {
+    private static GitHubApi instance;
+
+    private static void create() {
+
+      //------- enable logging ------
```
Comment (marcelpinto) [inline app/src/main/java/com/hardsoftstudio/rxflux/sample/core/GitHubApi.java:29]: enable logging?

File: app/src/main/java/com/hardsoftstudio/rxflux/sample/ui/mainactivity/MainActivity.java [hunk 3]
```diff
@@ -85,50 +82,44 @@ public void onRxStoreChanged(RxStoreChange change) {
...
     repositoriesStore.register();
     usersStore = UsersStore.get(SampleApp.get(this).getRxFlux().getDispatcher());
     usersStore.register();
   }
 
-  private void showUserDialog(String id) {
-    UserFragment dialog = UserFragment.newInstance(id);
-    dialog.show(getSupportFragmentManager(), UserFragment.class.getName());
+  private void showUserFragment(String id) {
+
```
Comment (marcelpinto) [inline app/src/main/java/com/hardsoftstudio/rxflux/sample/ui/mainactivity/MainActivity.java:116]: remove empty space

File: app/src/main/java/com/hardsoftstudio/rxflux/sample/ui/mainactivity/MainActivity.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package com.hardsoftstudio.rxflux.sample;
+package com.hardsoftstudio.rxflux.sample.ui.mainactivity;
```
Comment (marcelpinto) [inline app/src/main/java/com/hardsoftstudio/rxflux/sample/ui/mainactivity/MainActivity.java:1]: why do you create a package just for this activity? I am agree on creating the ui package but not the mainactivity

File: app/src/main/java/com/hardsoftstudio/rxflux/sample/ui/userfragment/UserFragment.java [hunk 5]
```diff
@@ -1,17 +1,19 @@
-package com.hardsoftstudio.rxflux.sample;
+package com.hardsoftstudio.rxflux.sample.ui.userfragment;
```
Comment (marcelpinto) [inline app/src/main/java/com/hardsoftstudio/rxflux/sample/ui/userfragment/UserFragment.java:1]: Same here. Move this to the ui package

**Comment by marcelpinto** [comment]
@AndreiD Thank you for the changes. They look pretty good.

Please fix the comments, then I will test it and merge it if everything works.

**Comment by AndreiD** [comment]
no time man... no time :(

**Comment by marcelpinto** [comment]
Well I don't think is that much work since you already did a lot, but if you can't do it I will do it by myself.

**Comment by marcelpinto** [comment]
Fix conflicts and comments in this PR https://github.com/skimarxall/RxFlux/pull/3

**Pull request closed without merging**